doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -698,7 +715,7 @@ PyUFunc_Type
           PyUFuncGenericFunction *functions;
           void **data;
           int ntypes;
-          int reserved1;
+          int version;
```

Comment:
I thought we were leaving this as `reserved1` (at least that is what is described below).

Reply:
reverted